examples/colab/tf2_hub_single_image_super_resolution_ESRGAN.ipynb

```diff
@@ -0,0 +1,459 @@
+{
```

Comment:
<p>Can you reexport the module to have the __call__ signature?</p><p> </p><p>Basically creating a tf.train.Checkpoint that has one function __call__, that just forwards its arguments to model.call.</p><p> </p><p>Then you will be able to do <code>model(lr_image)</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/hub/pull/352/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Looking into it.</p><p> </p>

Reply:
Well, I looked into it, the model inherits from tf.keras.models.Model. Which keeps the __call__ function to itself. I'm not sure if that can be overwritten by a custom function with a signature or not.